server/views/pages/prisons/sessionTemplates/addSessionTemplate.njk

```diff
@@ -422,6 +422,24 @@
               }
             ]
           }) }}
+
+          {{ govukCheckboxes({
+            name: "showPublicServices",
```

Comment:
I suppose a slightly more accurate name would be `showInPublicServices` - as it's not public services that are being shown; it's whether this template is shown **within them**